README.md

```diff
@@ -8,81 +8,82 @@
-  - **Free, powerful API** - Build scalable and fully automated human-in-the-loop machine learning pipelines with a powerful open API.
+* Support for all common Toloka use cases: creating projects, adding pools, uploading tasks, and so on.
+* Toloka entities are represented as Python classes. You can use them instead of accessing the API directly via HTTPS.
+* There’s no need to validate JSON files and work with them directly.
+* Support of both synchronous and asynchronous usages.
+* Streaming support: send and receive data to Toloka in real time. It’s useful when you need to pass data between two related projects: one for data labeling, and another for its validation.  
+* [AutoQuality](https://medium.com/toloka/automating-crowdsourcing-quality-control-ad057baf00fd) feature which automatically finds the best fitting quality control rules for your project.
 
-**⚠️ toloka-kit==1.0.0 breaking changes ⚠️**
+## Prerequisites
 
-Starting with 1.0.0 release only the core version of the package is installed by default. See "Optional dependencies"
-section for details.
+Before you begin, make sure that:
+* You are using [Python](https://www.python.org/) v3.7 or higher.
+* You are [registered](https://toloka.ai/docs/guide/concepts/access.html/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) in Toloka as a requester.
+* You have [topped up](https://toloka.ai/docs/guide/concepts/refill.html/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) your Toloka account.
+* You have [set up an OAuth token](https://toloka.ai/docs/api/concepts/access.html/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) to access Toloka API.
 
-Requirements
---------------
-- Python 3.7+
-- Register in [Toloka.ai](https://toloka.ai/?utm_source=github&utm_medium=site&utm_campaign=tolokakit) as requester. Registration process described [here](https://toloka.ai/en/docs/guide/concepts/access?utm_source=github&utm_medium=site&utm_campaign=tolokakit).
-- [Topping up your account](https://toloka.ai/en/docs/guide/concepts/refill.html?utm_source=github&utm_medium=site&utm_campaign=tolokakit).
-- Getting an OAuth. Learn more in [help](https://toloka.ai/en/docs/api/concepts/access?utm_source=github&utm_medium=site&utm_campaign=tolokakit) and in the image below.
-
-
-![How to get OAuth token](https://yastatic.net/s3/doc-binary/src/support/toloka/en/toloka-kit/learn-basics/get-oauth-token.png "How to get OAuth token")
-
-Get Started
---------------
-Installing toloka-kit is as easy as:
+## Get Started
+1. Install the Toloka-Kit package. Run the following command in the command shell:
 ```
 $ pip install toloka-kit
 ```
-Note: this project is still under heavy development and interfaces may change slightly. For production environments please specify exact package version. For example: `toloka-kit==0.1.26`
+For production environments, specify the exact package version. The latest stable version is 1.1.3.
```

Comment:
We will forget to update this for sure :) Let's reference PyPI for the latest version https://pypi.org/project/toloka-kit/